EIPS/eip-4844.md

```diff
@@ -45,10 +45,10 @@ Compared to full data sharding, this EIP has a reduced cap on the number of thes
 | `BLOB_COMMITMENT_VERSION_KZG` | `Bytes1(0x01)` |
 | `POINT_EVALUATION_PRECOMPILE_ADDRESS` | `Bytes20(0x14)` |
 | `POINT_EVALUATION_PRECOMPILE_GAS` | `50000` |
-| `MAX_DATA_GAS_PER_BLOCK` | `2**19` |
-| `TARGET_DATA_GAS_PER_BLOCK` | `2**18` |
+| `MAX_DATA_GAS_PER_BLOCK` | `786432` |
+| `TARGET_DATA_GAS_PER_BLOCK` | `393216` |
```

Comment:
checking math: 
2**18 works out to 2 blobs (4096 * 32 bytes per blob given trusted setup size)
2**19 works out to 4 blobs (max is double target)

if we increase target by 50% (to move from 2 to 3), then we in fact get the numbers proposed

Reply:
consider `3 * 2**18` and `3 * 2**17` for legibility?

Reply:
could add in the third column (so after the decimal representations), I do like having the multiplier * base factor representation

Reply:
the problem is that this depends on the specific context, e.g. not obvious why it always should be base 2. or also, might be better in this specific case to always use factor * 2**17 (so that factor would be the number of blobs)